documentation/source/users/rmg/running.rst

```diff
@@ -28,7 +28,7 @@ We recommend you make a job-specific directory for each RMG simulation. Some job
 
 
 Details on the multiprocessing implementation
-----------------------------------------------
```

Comment:
Any reason for the whitespace change?

Reply:
Not just whitespace. They're trying to make the underline the same length as the line it's underlining.